src/client/datascience/data-viewing/dataViewer.ts

```diff
@@ -174,16 +209,19 @@ export class DataViewer extends WebviewPanelHost<IDataViewerMapping> implements
             if (e instanceof JupyterDataRateLimitError) {
                 traceError(e);
                 const actionTitle = localize.DataScience.pythonInteractiveHelpLink();
-                this.applicationShell.showErrorMessage(e.toString(), actionTitle).then((v) => {
-                    // User clicked on the link, open it.
-                    if (v === actionTitle) {
-                        this.applicationShell.openUrl(HelpLinks.JupyterDataRateHelpLink);
-                    }
-                });
+                this.applicationShell.showErrorMessage(e.toString(), actionTitle).then(
```

Comment:
Feels like a goofy way to display errors `e.toString()`, existing code, hence no need to change.
I feel the UX is pretty poor in such cases, its basically dumping a stack trace to the user & they'd go, `what is this?`

Again, no need to change as its existing code.